Update es.po

materiagris opened the pull request:
Full translation corrected and updated

materiagris committed:
Update es.po

Full translation corrected and updated

materiagris committed:
Update es.po

Full translation corrected and updated

AJenbo commented:
I split up the commits and fixed the incorrect changes to how the source attributes had been changed.

@hiperiondev could you review this?

hiperiondev commented:
@materiagris Translations are very complex and always debatable, sorry for my doubts but I try to stay as faithful as possible to the original text.

materiagris commented:
> @materiagris Translations are very complex and always debatable, sorry for my doubts but I try to stay as faithful as possible to the original text.

Oh, no problem. I believe that debate always improves the quality of translation work, and therefore it is not only legitimate, it's the way.

I see the translation of this type of games as a living work that improves with each iteration, so when I opted for the less formal option, I did so because at that moment it seemed the most appropriate way to maintain consistency with the rest of the translations. However, I see no problem in keeping those that are most appropriate.

materiagris commented:
Let me defend some of the corrections I made in the translation work I have contributed, which are not only limited to optional or aesthetic issues.

The term “ear” is ‘oreja’ in Spanish and not “oído,” which refers to the sensory organ and not the ear.

I don't know exactly what “tronido” means when we talk about “thunderclap,” but I think “trueno” is the appropriate translation.

On the other hand, in Spanish, regardless of the local variant, only proper nouns should be capitalized: https://www.rae.es/dpd/may%C3%BAsculas

These are just a few examples, but there are more.

materiagris commented:
Thank you very much for your time and effort @hiperiondev. 

I try to respond to your objections here:

Automapa is not mapa. I don't know if is correct change this. _-OK, you are right. I was thinking that if there is just one map, maybe map was enough, I tried changing it and that was the last option I tried. Automapa is perfect._ 

-msgid "What arena do you want to visit?"
msgstr "¿Cuál arena quieres visitar?"
msgstr "¿Qué arena quieres visitar?"
Cual is correct because is a choice. _-I disagree with that use. It sounds as a foreign use of "cuál".  Cuál is better used if you are offering two or more explicit options referred in the same statement. For example: ¿Cuál de estas cosas prefieres? If you use Cuál with a single thing in the same statement it appears to be a concordance error._

use comando instead of orden is debatable
-msgid "Gives Arena Potions."
msgstr "Da pociones de arena."
msgstr "Permitir pociones en arena."
Why? _-Sounds better in my opinion. Sometimes the exact translation is not the best one. But that is not very important._

-#: Source/control.cpp:1342
msgid "Select current spell button"
msgstr "Seleccionar botón de hechizo actual"
msgstr "Botón de hechizo seleccionado"
Context? _-I don't remember why. Maybe I was trying an alternative not so long because in the game the space wasn't enough._

msgid "Toggle automap"
msgstr "Alternar mapa automático"
msgstr "Mostrar mapa automático"
toggle change option in 'not value' mode. Is not mostrar. There are more misused in translation. _-I think "mostrar" is more clear, when I read "alternar" I don't know with what are you "alternando". ;-P_

msgid "Party"
msgstr "Grupo"
msgstr "Evento"
Context? There are more uses of Party as Evento. Why? _Because I think a group is not a party in the way is used in the game._

msgid "Zoom"
msgstr "Ampliar"
msgstr "Zoom"
Why? _-Because Zoom is "Ampliar" and "Alejar" too, and is in the common use._

msgid "Pause Game (Alternate)"
msgstr "Pausar juego (alternativo)"
msgstr "Pausar juego (alternativamente)"
Context? _-It sounds much better in my opinion. Alternativo could be misunderstood as alternative and not as alternate._

msgid "Hold to set or use spell hotkeys."
msgstr ""
"Mantener presionado para configurar o usar las teclas de acceso rápido de "
"hechizos."
msgstr "Mantener presionado para establecer o usar los atajos para hechizos."
Missing hotkeys. Why? _I think "atajo de teclado" is better option as tecla de acceso rápido._

msgid "Riches abound when least expected"
msgstr "Las riquezas abundan cuando menos se espera"
msgstr "La riqueza abunda cuando menos se espera"
No needed. Plural is correct and "more poetic". _-OK. No problem with it, but to maintain concordance write "esperan" because it's plural. That's the reason because I changed the sentence._

#: Source/engine/demomode.cpp:215 Source/options.cpp:871
msgid "Rejuvenation Potion Pickup"
msgstr "Recoger poción de rejuvenecimiento"
msgstr "Recoger poción de rejuv."
Is it necessary to cut the word? _-Yes, if you don't cut it the text is to long in the game._

Source/gamemenu.cpp:71
msgid "Gamma"
msgstr "Gama"
msgstr "Gamma"
Not double m in Spanish. Correct translation is Gama. _-Gama could be misunderstood with others meanings of the word, and the greek form is well known to be referred as Gamma, the greek letter: https://dle.rae.es/gamma?m=form_

#: Source/items.cpp:1766
msgid "Right-click to view"
msgstr "Clic derecho para ver"
msgstr "clic derecho para ver"
_-Mostly aesthetic. Only sentences begin with capital and end with a dot. No dot, no capital._

#: Source/items.cpp:1769
msgid "Right-click to use"
msgstr "Clic derecho para usar"
msgstr "clic derecho para usar"
_-Mostly aesthetic. Only sentences begin with capital and end with a dot. No dot, no capital._

If is an independent paragraph must start with capital. Context?

#: Source/items.cpp:4015
msgid "quick attack"
msgstr "ataque rápido"
msgstr "ataque ágil"
Why? _Don't remember._

msgid "Skeleton King's Lair"
msgstr "Guarida del Rey Esqueleto"
msgstr "Guarida del rey esqueleto"
It's a proper noun, capital letters are needed. _You are right. I went too far here :-)_

"Stash version invalid. If you attempt to access your stash, data will be "
"overwritten!!"
msgstr ""
"Versión del guardado no válida. Si intentas acceder a él se sobrescribirán "
"los datos."
"Versión del alijo no válida. ¡Si intentas acceder a él se sobrescribirán los "
"datos!"
"alijo" is not a used word in Latinoamerica, please choise other more common.
_-I used that one as it's the word used in the Spanish version of Diablo II, and also known for it._

msgid "Frame Rate Control"
msgstr "Control de Velocidad de Cuadros"
msgstr "Control de FPS"
Why? _-Too long text and FPS is also known to be "fotogramas por segundo"._

"Manages frame rate to balance performance, reduce tearing, or save power."
msgstr ""
"Administra la velocidad de cuadros para equilibrar el rendimiento, reducir "
"el desgarro o ahorrar energía."
"Administra la velocidad de fotogramas para equilibrar el rendimiento, evitar "
"tearing o ahorrar energía."
Why? _-As before, "cuadro" versus "fotograma". I think "fotograma" is better option._

msgid "Keymapping"
msgstr "Mapeo de teclas"
msgstr "Teclado"
It should be "Mapa de teclas," not "Teclado" Context is needed. _-Teclado is enough to understand what you are doing. Map as verb doesn't work in common Spanish but in Chile._

-msgid "Padmapping"
msgstr "Mapeo del pad"
msgstr "Mando"
Idem. _-Idem as before. Pad is English, and mapeo doesn't exist in common Spanish, but in Chile._

msgid "Chance To Hit"
msgstr "Probabilidad de acertar"
msgstr "Prob. de acertar"
Why? _-Text too long, try to see the result in the game._

: Source/stores.cpp:1079
msgid "The Town Elder"
msgstr "El Sabio del Pueblo"
msgstr "Deckard Caín"
Why? _-I found the german translation better than El sabio del pueblo. Anyway whithout uppercase._

msgid "Theodore"
msgstr "Teodoro"
msgstr "Theodore"
Name translation was correct. Why change? _-Theodore is a proper name. Theodore is better as the Spanish version._

Source/translation_dummy.cpp:323
msgid "Falchion"
msgstr "Chafarote"
msgstr "Alfanje"
Is not alfanje: https://en.wikipedia.org/wiki/Falchion. I don't know correct translation. _-I used that because I think is the same used in later versions of Diablo. Anyway a "chafarote" is some kind of "alfanje"._

msgstr ""
"Maestro, tengo una experiencia extraña que contarle. Sé que tiene un gran "
"Maestro, tengo una experiencia extraña que contarte. Sé que tienes un gran "
In this dialogue, a formal tone should be used because he is addressing his teacher. (check this in other dialogues) Both are good to me. _-As you prefer. That's not very important, mostly aesthetic._

#: Source/translation_dummy.cpp:750
msgid "Look here... that's pretty funny, huh? Get it? Blind - look here?"
msgstr ""
"Mira aquí ... eso es muy gracioso, ¿eh? ¿Conseguirlo? Ciego, ¿mira aquí?"
msgstr "Mira aquí... eso es muy gracioso, ¿eh? ¿Lo pillas? Ciego, ¿Mira aquí?"
Translation is correct but "Lo pillas" is entirely Spanish from Spain. It would be preferable to use a more "neutral" expression.
_-If you think so I agree with you, but "¿Conseguirlo?" is not an option in my opinion. I changed that also because I wanted to eliminate the space before the dots there._

All spell to lower case? Why? -Only correct if is not a single line. Context? _Works better when you find a book of..., but that is mostly aesthetic._

materiagris commented:
About "Botón de hechizo seleccionado". That appears when the cursor is over the right button at the bottom.
<img width="1980" height="423" alt="Captura de pantalla 2025-11-14 102844" src="https://github.com/user-attachments/assets/5dc07f9c-e459-4dd8-9739-cd4bca815abe" />

materiagris committed:
Update es.po

Some corrections made about the terms automap, elder, party and some capitals letters.

materiagris commented:
About this issue:
> "Maestro, tengo una experiencia extraña que contarle. Sé que tiene un gran "
> "Maestro, tengo una experiencia extraña que contarte. Sé que tienes un gran "
> In this dialogue, a formal tone should be used because he is addressing his teacher. (check this in other dialogues) Both are good to me. -As you prefer. That's not very important, mostly aesthetic.
> 
> Then do not change. Same other parts in this intention of dialogues

I decided to maintain my first proposal. I revised again and I could see that the rest of the texts from the same character are in non formal sense. I think is annoying if the same character changes the style.

The rest of the changes are reverted in my last commit.

materiagris committed:
Update es.po

Minimal corrections

hiperiondev commented:
> About this issue:
> 
> > "Maestro, tengo una experiencia extraña que contarle. Sé que tiene un gran "
> > "Maestro, tengo una experiencia extraña que contarte. Sé que tienes un gran "
> > In this dialogue, a formal tone should be used because he is addressing his teacher. (check this in other dialogues) Both are good to me. -As you prefer. That's not very important, mostly aesthetic.
> > Then do not change. Same other parts in this intention of dialogues
> 
> I decided to maintain my first proposal. I revised again and I could see that the rest of the texts from the same character are in non formal sense. I think is annoying if the same character changes the style.
> 
> The rest of the changes are reverted in my last commit.

That would be incorrect. The entire conversation would need to be revised to give it a formal tone.

hiperiondev commented:
That's correct, all interactions with that character need to be corrected.

materiagris commented:
> That's correct, all interactions with that character need to be corrected.

Good, no problem with it. We can revise all the interactions and give him another style, but I think that would be made in the next iteration.

materiagris committed:
Update es.po

Some typos with spaces before dots corrected.

materiagris committed:
Update es.po and new iteration

Most of the dialogues are revised, correcting some errors. Odgen's dialogues are written in a more formal style.

Typos are corrected and some changes are made with special attention to verb tenses.

The term “Master” is clarified in each instance according to its different meanings: servant employer, slave owner, expert, or archaic title similar to Mister.

The inappropriate use of capital letters continues to be corrected.

Some texts are adapted for size reasons.

Some errors introduced in previous commits are reversed.

AJenbo commented:
> Anyway, I'd like to know @AJenbo 's opinion on how he considers the original tone of the English texts to be: Is the tone informal or formal? (understanding that it might be difficult to interpret in English)

It all depends, UI sould be neutral. Cain, Adria, Pepin are somewhat format. Odgen Farnhem, Wirt are informal. Lore is more dramatic in style.

Check what style Diablo II: Resurrected, Diablo III and Diablo I(mmortal) use in there official translations.

materiagris commented:
> > Anyway, I'd like to know @AJenbo 's opinion on how he considers the original tone of the English texts to be: Is the tone informal or formal? (understanding that it might be difficult to interpret in English)
> 
> It all depends, UI sould be neutral. Cain, Adria, Pepin are somewhat format. Odgen Farnhem, Wirt are informal. Lore is more dramatic in style.
> 
> Check what style Diablo II: Resurrected, Diablo III and Diablo I(mmortal) use in there official translations.

Thank you @AJenbo for your comments, I agree with you.

materiagris committed:
Update es.po

Reverted changes as we agreed.
Hit is now translated as 'impacto'.
More capital letters changed to be in Spanish way.
More corrections and text adaptations to fit well.
